index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>
```

Comment:
## Please take note of the three things I will list below; Kindly fix them.

- [ ] **Pull Request Title:** Pull Request should have a descriptive title telling you what changes are inside.
e.g Hello Microverse Project 
- [ ] **Pull request summary/description:** Pull Request should have a summary describing introduced changes in more detail using preferably bullet points like so 👇🏾 

```markdown
I implemented the following:
- Added HTML file with boilerplate code and `h1` with text "Hello Microverse".
- Mention something else you did
```
- [ ] The branch name you have here `html` is not descriptive, hence, not professional. Your branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section').  
- Please read these articles for ideas 
   - [Branch and commit convention](https://dev.to/varbsan/a-simplified-convention-for-naming-branches-and-commits-in-git-il4), 
   - [7 principles for branch naming](https://hackernoon.com/git-branch-naming-convention-7-best-practices-to-follow-1c2l33g2)

- [ ] Lastly, your commitment messages are not very professional. Please read this article to learn how to structure your commit messages. 
   - [Branch and commit convention](https://dev.to/varbsan/a-simplified-convention-for-naming-branches-and-commits-in-git-il4),  

![image](https://user-images.githubusercontent.com/84156714/219059383-7499ea15-0132-4f09-b191-5284b9dfd2e2.png)